src/components/uielements/SmallTile.astro

```diff
@@ -13,7 +13,7 @@ const {
 <div class:list={["inline-flex flex-col items-center justify-center w-full gap-8 px-12 py-6 border-blue-500 shadow backdrop-blur-lg lg:w-1/3 bg-gradient-to-l from-blue-600/70 to-blue-800/70 shadow-blue-200 rounded-2xl border-1", className]}>
     <slot />
     <div class="flex flex-col items-center justify-start gap-4">
-        <div class="font-serif text-xl text-center text-white" set:html={headline}></div>
-        <div class="text-base text-center text-white" set:html={description}></div>
+        <h4 class="font-serif text-xl text-center text-white" set:html={headline}></h4>
```

Comment:
Is er een reden waarom dit een h4 is? volgt het altijd op een h3?

Reply:
Goed punt, ik ga die configurable maken